fastlane/Fastfile

```diff
@@ -61,6 +61,58 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases and opens a PR"
+lane :open_pr_upgrading_dependencies do |options|
+  if options[:dry_run]
+    dry_run = true
+  end
+  
+  new_versions = update_native_dependencies_to_latest()
+  
+  repo_status = Actions.sh("git status --porcelain")
+
+  repo_clean = repo_status.empty?
+
+  if repo_clean
+    UI.success('Git status is clean, nothing to update! 💪')
+    next
+  end
+
+  new_ios_version = new_versions[:new_ios_version]
+  new_android_version = new_versions[:new_android_version]
+
+  branch = "update-#{new_ios_version}-#{new_android_version}"
+  
+  result = Actions.sh("git checkout -b #{branch}")
+  if result =~ /error:(.)/
+    UI.error("Git couldn't change to `#{branch}`. Reason: #{result}.")
+  else
+    UI.success("Git switched to branch `#{branch}, all good! 💪")
+  end
+  
+  result = Actions.sh("git commit -am \"Upgrades versions\"")
+  if result =~ /error:(.)/
+    UI.error("Git couldn't create commit. Reason: #{result}.")
+  end
+
+  if dry_run
+    UI.message("ℹ️  Nothing more to do, dry_run: true")
+  else
+    push_to_git_remote(
+      remote: "origin",
+      set_upstream: true
```

Comment:
Hmm are these parameters mandatory? I think these were the default but don't fully remember

Reply:
`remote: "origin",` is the default, I will remove that one. `set_upstream` defaults to `false`